lib/appsignal/probes/erlang_probe.ex

```diff
@@ -10,31 +10,45 @@ defmodule Appsignal.Probes.ErlangProbe do
 
   defp io_metrics do
     {{:input, input}, {:output, output}} = :erlang.statistics(:io)
-    @appsignal.set_gauge("erlang_io", Kernel.div(input, 1024), %{type: "input"})
-    @appsignal.set_gauge("erlang_io", Kernel.div(output, 1024), %{type: "output"})
+    set_gauge("erlang_io", Kernel.div(input, 1024), %{type: "input"})
+    set_gauge("erlang_io", Kernel.div(output, 1024), %{type: "output"})
   end
 
   defp scheduler_metrics do
-    @appsignal.set_gauge("erlang_schedulers", :erlang.system_info(:schedulers), %{type: "total"})
+    set_gauge("erlang_schedulers", :erlang.system_info(:schedulers), %{type: "total"})
 
-    @appsignal.set_gauge(
+    set_gauge(
       "erlang_schedulers",
       :erlang.system_info(:schedulers_online),
       %{type: "online"}
     )
   end
 
   defp process_metrics do
-    @appsignal.set_gauge("erlang_processes", :erlang.system_info(:process_limit), %{type: "limit"})
+    set_gauge("erlang_processes", :erlang.system_info(:process_limit), %{type: "limit"})
 
-    @appsignal.set_gauge("erlang_processes", :erlang.system_info(:process_count), %{type: "count"})
+    set_gauge("erlang_processes", :erlang.system_info(:process_count), %{type: "count"})
   end
 
   defp memory_metrics do
     memory = :erlang.memory()
 
     Enum.each(memory, fn {key, value} ->
-      @appsignal.set_gauge("erlang_memory", Kernel.div(value, 1024), %{type: to_string(key)})
+      set_gauge("erlang_memory", Kernel.div(value, 1024), %{type: to_string(key)})
     end)
   end
+
+  defp set_gauge(name, value, tags) do
+    @appsignal.set_gauge(
+      name,
+      value,
+      Map.merge(tags, %{host_metric: "", hostname: hostname()})
```

Comment:
Let's remove the `host_metric: ""`. It's not a host_metric. It's a host specific metric, but not a [host metric](https://docs.appsignal.com/metrics/host.html)

Reply:
Hang on, now I'm confused. This means you're getting back on what you said earlier, correct?

Reply:
Oh wait, after reading back your comment I realise it is about the `void appsignal_set_host_gauge(appsignal_string_t, double);` suggestion.